docs/env-vars.md

```diff
@@ -4,31 +4,34 @@
+| `IPFS_CONTEXT_PATH`                     | IPFS Context Path                                                                                                                                                                                     | `string`               | `'/ipfs/'`                  |
+| `IPFS_USER_AGENT_VALUE`                 | Default user agent that is going to be used for public IPFS queries.                                                                                                                                  | `string`               | `'ewx-worker-node-server'`  |
+| `SOLUTION_QUEUE_PROCESS_DELAY`          | How often should refresh EWX Solutions information from chain (in miliseconds).                                                                                                                       | `number` (_nullable_)  | `20000`                     |
+| `LOCAL_SOLUTIONS_PATH`                  | Path to locally hosted NodeRed solution flow files to be used when installing solutions using "local" prefix within WorkLogic field.                                                                  | `string`               |                             |
+| `SS58_FORMAT`                           | SS58 Key Format.                                                                                                                                                                                      | `number` (_>0_)        | `42`                        |
+| `PRETTY_PRINT`                          | Should pretty print logs. If you plan to use Grafana or any other log tooling it's recommended to set it to false.                                                                                    | `'true' \| 'false'`    | `'false'`                   |
+| `HEARTBEAT_PATH`                        | Path to the heartbeat file used for monitoring.                                                                                                                                                       | `string`               | `'heartbeat'`               |
+| `HEARTBEAT_INTERVAL`                    | Interval (in ms) at which the heartbeat process updates the file.                                                                                                                                     | `number` (_>0_)        | `5000`                      |
+| `HEARTBEAT_PRINT_SUCCESS_LOG`           | Should print successful logs.                                                                                                                                                                         | `'true' \| 'false'`    | `'true'`                    |
+| **`PALLET_AUTH_SERVER_LOGIN_URL`** (\*) | Pallet Auth Server URL                                                                                                                                                                                | `string` (_url_)       |                             |
```

Comment:
Description:

"Pallet Auth Server Url used for authentication for Workers Registry"